docusaurus/docs-internals/usage-in-html/usage.md

```diff
@@ -0,0 +1,675 @@
+---
+sidebar_position: 1
+toc_max_heading_level: 4
+---
+
+# Examples
+Brizy's control values can be used to render content in the HTML. These values enable you to dynamically display text, images, and other elements within your web pages
+
+
+### AiText
+In the example below, the value of the `aiText` control allows us to render the generated text in the HTML.
+
+```tsx
+import { Brizy } from "@brizy/core";
+import React, { JSX } from "react";
+
+interface Props {
+  text: string
+}
+
+const GeneratedText = (props: Props): JSX.Element => {
+  const { text } = props;
+  
+  return (
+    <div className="brz-ai-text">
+      <h3>Your generated text by AI :</h3>
+      <span>{text}</span>
+    </div>
+  );
+};
+
+Brizy.registerComponent(GeneratedText, {
+  id: "ThirdParty.GeneratedText",
+  title: "Ai Text",
+  options: (props) => {
+    return [
+      {
+        selector: ".brz-ai-text",
+        toolbar: [
+          {
+            id: "aiText",
+            type: "popover",
+            config: {
+              icon: "t2-star-shapes",
+              size: "auto",
+              title: "Ai Text"
+            },
+            options: [
+              {
+                id: "text",
+                type: "aiText"
+              }
+            ]
+          }
+        ]
+      }
+    ];
+  }
+});
```

Comment:
review registering of your example because now registering look like this:
![image](https://github.com/EasyBrizy/Brizy-Local-Editor/assets/52622998/e1b204b5-94de-4e41-abe1-5ed2440ea211)